### DISCUSSION TITLE
Is the ABI guaranteed to the be same for `MAJOR.MINOR.*`

### DISCUSSION THREAD

**Discussion by hadim** [rdkit/rdkit#4635] [Q&A] [answered]
@greglandrum I am trying to build a conda forge package for a lib that depends on the rdkit C++ ABI. I would like to build it against the last three major.minor versions of RDKit ideally but I need to know whether the ABI is guaranteed to be identical for the same `MAJOR.MINOR` versions.

See https://github.com/conda-forge/staged-recipes/pull/16618#pullrequestreview-787053200 for more context.

Thanks.

**Comment by hadim** [comment]
The lib is on conda-forge now. See the ticket https://github.com/conda-forge/rdchiral_cpp-feedstock/issues/1

**Comment by greglandrum** [comment, marked as answer]
Since we just do bug fixes with patch releases, the ABI should not change
